src/app/lib/config/services/polska.ts

```diff
@@ -330,7 +329,7 @@ export const service: DefaultServiceConfig = {
       },
     },
     mostRead: {
-      header: 'Most read',
+      header: 'Popular Reads',
```

Comment:
So, `most read` isn't being renamed to `popular reads`? the way it reads here is confusing (not what you've done)

Reply:
Yea can see why its a bit confusing. Most Read as a concept isn't changing names, just the title of the component at the bottom of the pages will be changing to 'Popular Reads'.

Reply:
Cool, this might be one where we have to migrate our language to using the same terminology so it doesn't confuse newbies